odoo-shippable/scripts/build-image.sh

```diff
@@ -129,6 +129,18 @@ wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/ru
 sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
 sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
 
+# Install YouCompleteMe
+apt-get install cmake
```

Comment:
@ruiztulio  What do you think about updating the vim at https://github.com/Vauxoo/docker-ubuntu-base/blob/master/scripts/build-image.sh#L31
According to a comment by @LuisAlejandro  https://github.com/Vauxoo/docker-odoo-image/pull/174#discussion_r96301022